test/test_meta.py

```diff
@@ -508,6 +508,7 @@ def run_meta_crossref(
     torch.equal : {bf16, i8, c32, i64, u8, c128, b8, f64, i16, i32, f32, f16, c64},
     torch.functional.cdist : {f64, f32},
     torch.nanmean : {bf16, f64, f32, f16},
+    torch.logical_not : {f32},
```

Comment:
Why is this skip necessary? I don't see how this PR affects meta test

@khushi-411 can you please take a look

Reply:
Yep, I agree with you! As far as I remember, I think there were some CI failures at that time. I've undone the changes to re-check it. Thanks for pointing it out, @kshitij12345.